src/main/java/net/fabricmc/loom/api/LoomGradleExtensionAPI.java

```diff
@@ -82,6 +83,7 @@ default Dependency officialMojangMappings() {
 
 	NamedDomainObjectContainer<RunConfigSettings> getRunConfigs();
 
+	@ApiStatus.Experimental
 	void mixin(Action<MixinApExtension> action);
```

Comment:
Shouldn't this accept the API?

Reply:
True. I forget this part when I do the refactor.

Reply:
updated.